src/org/exist/xmldb/RemoteResourceSet.java

```diff
@@ -235,7 +235,8 @@ public void removeResource(final long pos) throws XMLDBException {
     protected void finalize() throws Throwable {
         try {
             clear();
-        } finally {
+        }
```

Comment:
what formatting rules do you use? i think in the code base `} finally {' is done on most locations....